packages/vite-plugin-react-docgen-typescript/src/index.ts

```diff
@@ -30,38 +60,38 @@ const getUtils = async (config: Options) => {
 		)
 		.reduce((carry, files) => carry.concat(files), []);
 
-	const tsProgram = ts.createProgram(files, compilerOptions);
-
-	const result = {
-		docGenParser,
-		filter,
-		generateOptions,
-		generateDocgenCodeBlock,
-		tsProgram,
-	};
-
-	return result;
+	return ts.createProgram(files, compilerOptions, undefined, oldProgram);
```

Comment:
Hey @joshwooding 

What happens if we pass the `oldProgram` to the `ts.createProgram` function? Is Typescript able to speed up the new program's creation?

Reply:
Yeah, that's in theory how it works